.github/workflows/pb-update-go.yml

```
@@ -1,7 +1,7 @@
 name: Update Go
 "on":
     schedule:
-        - cron: 32 2 * * 1
+        - cron: 27 2 * * 1
```

```suggestion
        - cron: 32 2 * * 1
```

Changes to this are not usually made expected. Was there a reason?